network-api/networkapi/wagtailpages/templates/wagtailpages/primary_page.html

```diff
@@ -1,13 +1,19 @@
-{% extends "pages/base-compiled.html" %} {% load wagtailcore_tags wagtailimages_tags %}
+{% extends "./modular_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags primary_page_tags %}
 
-{% block bodyID %}primary{% endblock%}
+{% block bodyID %}{{ page.title }}{% endblock%}
```

Comment:
The main goal of this tag is for CSS, so I think it's better to keep this to the template name rather than a piece of CMS data. Was there a specific reason for this change?